educator/src/Dscp/Educator/Web/Server.hs

```diff
@@ -49,35 +52,34 @@ mkStudentApiServer
     -> m (Server ProtectedStudentAPI)
 mkStudentApiServer nat botSwitch = do
     case botSwitch of
-      EducatorBotOff -> return $ addAuth (getServer . studentApiHandlers)
+      EducatorBotOff -> return $ getServer . studentApiHandlers
       EducatorBotOn params -> initializeBot params $ do
-        return . addAuth $ (\student -> getServer . addBotHandlers student . studentApiHandlers $ student)
+        return $ (\student -> getServer . addBotHandlers student . studentApiHandlers $ student)
   where
     getServer handlers = hoistServerWithContext
         studentAPI
-        (Proxy :: Proxy '[CookieSettings, JWTSettings, GetStudentsAction])
+        (Proxy :: Proxy '[GetStudentsAction])
         nat
         (toServant handlers)
-    addAuth :: (Student -> Server StudentAPI) -> Server ProtectedStudentAPI
-    addAuth h = \case
-        Authenticated (WithStudent student _) -> h student
-        _ -> throwAll err401
+
+-- This is a temporary function that provides a dummy GetStudentAction
+createGetStudentsAction :: IO GetStudentsAction
```

Comment:
How do we know which secret keys were generated? Post them somewhere maybe?

Reply:
This change is part of DSCP-141 which is still not merged yet. Here I just separated the key generation into its own function as @Martoon-00 requested, and instead of generating just two I generate 100. Two of them are posted in #120 description.